cmd/epp/runner/runner.go

```diff
@@ -397,6 +392,31 @@ func (r *Runner) Run(ctx context.Context) error {
 	return nil
 }
 
+func setupDataStore(setupLog logr.Logger, ctx context.Context, epFactory datalayer.EndpointFactory, modelServerMetricsPort int32, gknn common.GKNN) (datastore.Datastore, error) {
+	if gknn.Kind == "InferencePool" {
+		return datastore.NewDatastore(ctx, epFactory, modelServerMetricsPort), nil
+	}
+	if gknn.Kind == "Deployment" {
```

Comment:
I find this confusing, gknn represents the name and kind of a k8s resource, why are we using that to encode how the endpoints are discovered?